CHANGELOG_PENDING.md

```diff
@@ -70,14 +72,14 @@ program](https://hackerone.com/tendermint).
 - Blockchain Protocol
 
   - [abci] \#2521 Remove `TotalTxs` and `NumTxs` from `Header`
-  - [types] [\#4151](https://github.com/tendermint/tendermint/pull/4151) Enforce ordering of votes in DuplicateVoteEvidence to be lexicographically sorted on BlockID
+  - [types][\#4151](https://github.com/tendermint/tendermint/pull/4151) Enforce ordering of votes in DuplicateVoteEvidence to be lexicographically sorted on BlockID
```

Comment:
🤣 so I am adding white spaces and you're deleting them, nice

Reply:
ah sorry this is my editor, will turn it off